requirements/base.txt

```diff
@@ -40,20 +41,12 @@ django==2.2.24
     #   edx-auth-backends
     #   edx-django-release-util
     #   edx-django-utils
-django-cors-headers==2.2.1
-    # via
-    #   -c requirements/constraints.txt
-    #   -r requirements/base.in
 django-crum==0.7.9
     # via edx-django-utils
-django-environ-2==2.1.0
-    # via
-    #   -c requirements/constraints.txt
-    #   -r requirements/base.in
-django-filter==2.1.0
-    # via
-    #   -c requirements/constraints.txt
-    #   -r requirements/base.in
+django-environ==0.7.0
```

Comment:
This was originally [pinned](https://github.com/edx/blockstore/pull/121/files#diff-86d5fe588ff2fc7dccb1f4cdd8019d4473146536e88d7a9ede946ea962a91acbL23) to v.0.4.5

There don't seem to be any backwards incompatible changes between 0.4.5 and 0.7.0: https://github.com/joke2k/django-environ/blob/main/CHANGELOG.rst